celery_status_server.py

```diff
@@ -0,0 +1,110 @@
+#celery_status_server.py
+
+from flask import Flask, request, make_response
+from celery import Celery
+from celery.app.control import Inspect
+import datetime
+import requests
+import os
+import json
+import time
+import sys
+import subprocess as sp
+from functools import partial
+import example_async
+"""
+nohup redis-server &
+nohup celery -A example_async worker -P eventlet -l info &
+"""
+server_app = Flask('celery_status_server')
+port = 5070
+server_url = "http://localhost:{0}".format(port)
+
+TICKET_IDS_WATCHED = {}
+m = '--module'
+if not m in sys.argv:
+    raise ValueError('needs --module that has the celery_app importable from here')
+else:
+    celery_app = __import__(sys.argv[sys.argv.index(m) + 1]).__dict__['celery_app']
+
+def inspect_celery():
+    global TICKET_IDS_WATCHED
+    return_val = {}
+    to_pop = []
+    i = celery_app.control.inspect()
+    print('i',i)
+    for label, worker_check in zip(('active', 'scheduled'),(i.active(), i.scheduled())):
+        print('worker_check', worker_check)
+        if worker_check:
+            for worker_id, worker_dict_list in worker_check.items():
+
+                for worker_dict in worker_dict_list:
+                    request = worker_dict.get('request', {})
+                    job_id = worker_dict.get('id', request.get('id', None))
+                    eta = worker_dict.get('eta', None)
+                    if job_id in TICKET_IDS_WATCHED:
+                        return_val[worker_dict['id']] = {'inputs': TICKET_IDS_WATCHED[job_id].copy(),
+                                                         'status': label,
+                                                         'eta': eta,
+                                                         }
+                    else:
+                        pass
+    for key in TICKET_IDS_WATCHED:
+        if key not in return_val:
+            return_val[key] = {'inputs': TICKET_IDS_WATCHED[key].copy(),
+                               'status': 'done'}
+            to_pop.append(key)
+    response = {}
```

Comment:
It should be possible here to call AsyncResult() to see if the job is really done or if the ticket id was never processed.  Currently either of those cases would show up here as 'done' status.